app/src/main/java/com/hamdan/forzenbook/view/login/CreateAccount.kt

```diff
@@ -259,45 +328,40 @@ private fun Content(
         imeAction = ImeAction.Done,
         onDone = {
             keyboardController?.hide()
-            if (!(emailError || birthError || locationError || lastNameError || firstNameError)) {
+            if (!(emailError != EmailErrors.VALID || birthError != BirthDateErrors.VALID || locationError != LocationErrors.VALID || lastNameError != NameErrors.VALID || firstNameError != NameErrors.VALID)) {
```

Comment:
Make extension functions `isValid` so you don't need these verbose checks

Reply:
created as a sealed interface function fun times